apps/web/pages/api/book/event.ts

```diff
@@ -508,7 +508,27 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       }
     }
 
-    return prisma.booking.create(createBookingObj);
+    /*Validate if there is any stripe_payment credential for this user*/
```

Comment:
`/* Validate if there is any stripe_payment credential for this user */`
Missing spaces after, before comment code